hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/resources/container-log4j.properties

```diff
@@ -26,36 +26,31 @@ log4j.threshold=ALL
 #
 
 #Default values
-yarn.app.container.log.dir=null
-yarn.app.container.log.filesize=100
+yarn.app.container.log.filesize=100MB
 
-log4j.appender.CLA=org.apache.hadoop.yarn.ContainerLogAppender
-log4j.appender.CLA.containerLogDir=${yarn.app.container.log.dir}
-log4j.appender.CLA.containerLogFile=${hadoop.root.logfile}
-log4j.appender.CLA.totalLogFileSize=${yarn.app.container.log.filesize}
+log4j.appender.CLA=org.apache.log4j.RollingFileAppender
+log4j.appender.CLA.File=${yarn.app.container.log.dir}/${hadoop.root.logfile}
+log4j.appender.CLA.MaxFileSize=${yarn.app.container.log.filesize}
```

Comment:
We should also config MaxBackupIndex? What is the default value if we do not set it?

Reply:
The default value of MaxBackupIndex is 1 in RFA

Reply:
Done